contracts/delegate.sol

```diff
@@ -230,15 +230,14 @@ contract JuiceBuyback is IJBFundingCycleDataSource, IJBPayDelegate, IUniswapV3Sw
     (, , uint256 _quote, uint256 _slippage) = abi.decode(_data.metadata, (bytes32, bytes32, uint256, uint256));
     uint256 _minimumReceivedFromSwap = _quote * _slippage / SLIPPAGE_DENOMINATOR;
 
-    // Pick the appropriate pathway (swap vs mint)
-    if (_minimumReceivedFromSwap > _tokenCount) {
+    // Pick the appropriate pathway (swap vs mint), use mint if non-claimed prefered
+    if (_minimumReceivedFromSwap > _tokenCount || !_data.preferClaimedTokens) {
```

Comment:
instead of this `if (_minimumReceivedFromSwap > _tokenCount || !_data.preferClaimedTokens)` probably
`if (_minimumReceivedFromSwap > _tokenCount && _data.preferClaimedTokens)` wdyt ?